Postman/README.md

```diff
@@ -0,0 +1,23 @@
+## Postman
+
+Postman is a tool for working with APIs. It has a lot of nice features: 
+
+* Collections: group API requests together so you find them quickly (instead of scrolling through the History list)
+* Environments: easily switch between production, live and development environments
+* Variables: store the AuthenticationToken from SignIn request and store it in a variable so you can use it in other API requests. This saves you a lot of annoying copy-pasting. 
+* Tests: check status codes and verify responses (e.g. check if all surveys have a name)
+* Automate: run a collection for repetitive tasks (sign in, create survey, upload script, publish survey, start fieldwork)
+* Generate code: postman can even generate code based on a request
+
+![Postman with Nfield API Collection](Postman.png)
+*Figure 1. Postman with Nfield API Collections*
```

Comment:
It would be better if the screenshot showed one of the real environments instead of localhost

Reply:
ok